CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 All notable changes to the DTS project will be documented in this file.
 
+## v1.2.7 - 2023-09-05
+
+* Added Dasharo zero-touch initial deployment on MSI PRO Z790-A for
+  [DES](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
+  users.
+* Added Dasharo firmware update to version v1.2.0 on MSI PRO Z690-A for
```

Comment:
1.2.0, not 1.1.2?

Reply:
@macpijan typo, fixed https://github.com/Dasharo/meta-dts/pull/36/commits/22d0f8dd62460cca1c567824392c4c006f520308#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10